### PR TITLE
Add title and description to Legend

### PR DESCRIPTION
**Pull request by hojelse**
Resolves: #173

### CODE REVIEW

File: src/metrics.ts [hunk 1]
```diff
@@ -10,13 +10,30 @@ import { getColorFromExtension } from "./extension-color"
...
+export function getMetricDescription(metric: MetricType): string {
+  switch (metric) {
+    case "FILE_EXTENSION":
+      return "Where are different types of files located?"
+    case "MOST_COMMITS":
+      return "Which files have had the most commits, throughout the repository's history?"
+    case "LAST_CHANGED":
+      return "Where are the most recent or least recent commits made?"
+    case "SINGLE_AUTHOR":
+      return "Which files are authored by only 1 person, throughout the repository's history?"
```
Comment (emiljapelt) [inline src/metrics.ts:29]: change "1" to "one"

**Review by emiljapelt — DISMISSED** [review]
Also, maybe relative for "Last Changes" metric

File: src/metrics.ts [hunk 2]
```diff
@@ -10,13 +10,30 @@ import { getColorFromExtension } from "./extension-color"
...
+    case "TOP_CONTRIBUTOR":
+      return "Which person have made the most line-changes to a file, throughout the repository's history?"
```
Comment (tjomson) [inline src/metrics.ts:31]: ~~have~~ -> has

**Review by joglr — APPROVED** [review]
🚀

**Pull request merged**